dlib/dnn/cpu_dlib.cpp

```diff
@@ -1699,6 +1699,28 @@ namespace dlib
             }
         }
 
+        void tensor_conv::forward_without_setup(
```

Comment:
This code looks the same as tensor_conv::operator().  Am I just not seeing it?  It looks identical.